chantal-sockets

```diff
@@ -0,0 +1,9 @@
+def binary_to_decimal(binary_array)
+  raise NotImplementedError
```

Comment:
Delete this line since you're now implementing the method and you'll need this removed for the tests to pass.